client/src/scenes/Game.ts

```diff
@@ -143,35 +182,59 @@ export default class Game extends Phaser.Scene {
             }
         });
 
+
+
+
+    }
+
+
+
+    // set up the map and the different layers to be added in the map for reference in collisionSetUp
+    private mapTileSetUp() {
+
+
         const map = this.make.tilemap({ key: 'user_room' })
```

Comment:
why cant we use this map directly as a field?
this.map
not sure about whether it supports retrieval of layer

Reply:
will look into this